ci/add-changelog-entry.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# This is a simple interactive script for adding changelog entries.
+
+set -euo pipefail
+
+function get_default_change_type() {
+    local text
+    readonly text="${1}"
+    readonly commit_type="${text%[(:]*}"
+
+    local CHANGE_TYPE 
+    case "${commit_type}" in
+    fix )
+        CHANGE_TYPE="fixed"
+        ;;
+    feat )
+        CHANGE_TYPE="added"
+        ;;
+    * )
+        CHANGE_TYPE="changed"
```

Comment:
What about breaking changes? Shouldn't there be a separate category for it?

Reply:
This just tries to infer the change type from commit title and sets that as the default. You can still set the change as breaking. Though I suppose I could check if the commit prefix has an exclamation mark and set based on that.